projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/checkout-config.service.migration.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  CHECKOUT_CONFIG_SERVICE,
+  CHECKOUT_STEP_SERVICE,
+  GET_CHECKOUT_STEP,
+  GET_CHECKOUT_STEP_ROUTE,
+  GET_CURRENT_STEP_INDEX,
+  GET_FIRST_CHECKOUT_STEP_ROUTE,
+  GET_NEXT_CHECKOUT_STEP_URL,
+  GET_PREVIOUS_CHECKOUT_STEP_URL,
+  SPARTACUS_STOREFRONTLIB,
+  STEP,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+// projects/storefrontlib/src/cms-components/checkout/services/checkout-config.service.ts
+export const CHECKOUT_CONFIG_SERVICE_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: STEP,
+    comment: `// ${TODO_SPARTACUS} Method '${STEP}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use ${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_CHECKOUT_STEP,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_CHECKOUT_STEP}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_CHECKOUT_STEP}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_CHECKOUT_STEP_ROUTE,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_CHECKOUT_STEP_ROUTE}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_CHECKOUT_STEP_ROUTE}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_FIRST_CHECKOUT_STEP_ROUTE,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_FIRST_CHECKOUT_STEP_ROUTE}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_FIRST_CHECKOUT_STEP_ROUTE}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_NEXT_CHECKOUT_STEP_URL,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_NEXT_CHECKOUT_STEP_URL}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_NEXT_CHECKOUT_STEP_URL}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_PREVIOUS_CHECKOUT_STEP_URL,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_PREVIOUS_CHECKOUT_STEP_URL}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_PREVIOUS_CHECKOUT_STEP_URL}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+  {
+    class: CHECKOUT_CONFIG_SERVICE,
+    importPath: SPARTACUS_STOREFRONTLIB,
+    deprecatedNode: GET_CURRENT_STEP_INDEX,
+    comment: `// ${TODO_SPARTACUS} Method '${GET_CURRENT_STEP_INDEX}' was removed from '${CHECKOUT_CONFIG_SERVICE}'. Instead use new method '${GET_CURRENT_STEP_INDEX}' from '${CHECKOUT_STEP_SERVICE}'.`,
+  },
+];
```

Comment:
Going to put it out there, but what about the 3 private methods that were removed 
`getStepUrlFromActivatedRoute`
`getStepUrlFromStepRoute`
`getCheckoutStepIndex`

Reply:
Should we include private methods here since customers cannot (should not) be using them in 2.x anyway?

Reply:
true, was just putting out there just in case